### PR TITLE
Story/nfg 86 create inventory items

### PR DESCRIPTION
**Pull request by nfgallimore**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -3,7 +3,11 @@
   <head>
     <meta charset="utf-8" />
     <title>Nufacturing Elite Manager</title>
+    <<<<<<< Updated upstream
     <base href="/nufacturing/" />
+    =======
+    <base href="C:/Program Files/Git/nufacturing/" />
+    >>>>>>> Stashed changes
```
Comment (nfgallimore) [inline docs/index.html:10]: Merge conflict

File: docs/BUG_CATEGORIES.md
Comment (nfgallimore) [inline docs/BUG_CATEGORIES.md:1]: Delete

File: README.md [hunk 2]
```diff
@@ -238,4 +238,4 @@ Use tags for marking significant releases or milestones. For example, `v1.0.0`.
 
 By maintaining these five environments, Gallimore Software ensures a robust, reliable, and efficient development and deployment process for the Nufacturing app. This approach leads to higher quality software and improved user satisfaction, aligning with our goals of delivering innovative and customer-centric solutions.
 
-[def]: ./docs/workflow.png
+[def]: ./docs/WORKFLOW.png
```
Comment (nfgallimore) [inline README.md:241]: Remove

File: src/app/app.module.ts [hunk 3]
```diff
@@ -41,6 +40,11 @@ import { AuthService } from './components/auth/auth.service';
 import { AuthGuard } from './components/auth/auth.guard';
 import { JWT_OPTIONS, JwtHelperService } from '@auth0/angular-jwt';
 import { AuthInterceptor } from './components/auth/auth.interceptor';
+import { ProductDevelopmentModule } from './pages/product-development/product-development.module';
+
+// Import the new SyncStatus components and services
```
Comment (nfgallimore) [inline src/app/app.module.ts:45]: Remove comments

File: src/app/app.module.ts [hunk 4]
```diff
@@ -52,6 +56,8 @@ import { AuthInterceptor } from './components/auth/auth.interceptor';
     RegisterComponent,
     LogoutComponent,
     LogoutDialogComponent,
+    // Add SyncStatusComponent
```
Comment (nfgallimore) [inline src/app/app.module.ts:59]: Remove comments

File: src/app/app.module.ts [hunk 5]
```diff
@@ -85,6 +91,7 @@ import { AuthInterceptor } from './components/auth/auth.interceptor';
   providers: [
     AuthService,
     AuthGuard,
+    SyncStatusService, // Add SyncStatusService here
```
Comment (nfgallimore) [inline src/app/app.module.ts:94]: Remove comments

**Review by Ahim-Gillamore — APPROVED** [review]
TODO check index.html

**Review by chrisatgallimore — APPROVED** [review]
PR Approved! Thanks!

**Pull request merged**